ocean/diagnostic_profiles/source_yaml_files/diag_table_standard_source.yaml

```diff
@@ -335,22 +336,31 @@ diag_table:
 
     'monthly 2d BGC fields':
         defaults:  # these can be overridden for individual fields below
+            module_name: generic_wombatlite
             file_name_prefix: oceanbgc
             file_name_dimension: 2d  # descriptor for filename, e.g. 3d, 2d, scalar
             output_freq: 1  # integer: output sampling frequency in output_freq_units (0: every timestep; -1: only at end of run)
             output_freq_units: months  # time units for output: years, months, days, hours, minutes, or seconds
         fields:
-            stf03:
-            stf07:
-            stf09:
-            stf10:
+            o2_stf:
+            dic_stf:
+            adic_stf:
```

Comment:
This isn't right, but fine for now. We will need to update this diagnostics file properly at a later date.

Reply:
Thanks Pearse, I'll make an issue for fixing these and removing the `make_wombatlite_restart.py` file